src/pal/src/misc/miscpalapi.cpp

```diff
@@ -40,6 +40,14 @@ Revision History:
 #include <mach-o/dyld.h>
 #endif // __APPLE__
 
+#ifdef HAVE_SYS_GETRANDOM
```

Comment:
This needs to be `#if HAVE_SYS_GETRANDOM` as I have mentioned in my previous comment.

Reply:
Done